attributes/default.rb

```diff
@@ -24,3 +24,4 @@
   bind: '0.0.0.0'
 }
 default["td_agent"]["yum_amazon_releasever"] = "$releasever"
+default['td_agent']['internal_repository'] = false
```

Comment:
Could you consider rephrasing the attribute name of `internal_repository`....? I can understand the purpose of it, but what it actually is "skip setting up repository". In general, I prefer naming a _feature_ after what it is doing. How do you feel with `default['td_agent']['skip_repository']` or something like that?

Reply:
that works for me, I'll make the change